md/release-notes.md

```diff
@@ -1,44 +1,38 @@
 # Release notes
 
-Bonita BPM 7.5 includes a set of new features related to the Living Application development and deployment and UI personalization in addition to important technology updates and bug fixes.
+Bonita BPM 7.5 includes a set of new features related to Living Application development and deployment as well as UI personalization, in addition to important technology updates and bug fixes.
```

Comment:
The product renaming will only occur from Bonita 7.6 and above.
This PR is about 7.5 release note.